### PR TITLE
add more godep notes

### PR DESCRIPTION
**Pull request by natebrennand**

### CODE REVIEW

**Comment by natebrennand** [comment]
CC @alsmola

File: golang/godep.md [hunk 1]
```diff
@@ -110,6 +112,55 @@ godep: no packages can be updated
 $ godep update github.com/Clever/worker/...
 ```
 
+## How do I do X?
+
+### first vendor save
```
Comment (rgarcia) [inline golang/godep.md]: nit: is there a better way to phrase this? "How do I do first vendor save" feels off

File: golang/godep.md [hunk 2]
```diff
@@ -110,6 +112,55 @@ godep: no packages can be updated
 $ godep update github.com/Clever/worker/...
 ```
 
+## How do I do X?
```
Comment (rgarcia) [inline golang/godep.md]: nit: maybe change this to "How do I ..." and change "adding" -> "add" and "updating" -> "update" below

**Comment by rgarcia** [comment]
lgtm. You should consider contributing something like this to godep's readme 😄

**Pull request merged**